### ISSUE TITLE
BackupWorker can't generate archive if there are local-only statuses

### ISSUE THREAD

**Issue by wxcafe** [glitch-soc/mastodon#1340]
When a user tries to generate an archive, it fails with this sidekiq traceback (archive button returns to active afterwards). It seems to be related to local-only statuses

```
{
  "context": "Job raised exception",
  "job": {
    "retry": 5,
    "queue": "pull",
    "backtrace": true,
    "dead": false,
    "class": "BackupWorker",
    "args": [
      21
    ],
    "jid": "8a2071be32b2ae0726611d44",
    "created_at": 1590680169.3343077,
    "enqueued_at": 1590680477.3722036,
    "error_message": "Local-only statuses should not be serialized",
    "error_class": "Mastodon::NotPermittedError",
    "failed_at": 1590680170.714574,
    "retry_count": 3,
    "error_backtrace": "[...]",
    "retried_at": 1590680344.7813993
  },
  "jobstr": "[...]"
}

```

* * * *

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [???] This bugs also occur on vanilla Mastodon

**Comment by ThibG** [comment]
Are you up to date? It should have been fixed last week (#1332)

**Comment by wxcafe** [comment]
im not, time to update I guess. Thanks! :heart:
